euth_wagtail/settings/base.py

```diff
@@ -318,6 +318,7 @@
     'avatar': {'min_resolution': (200, 200)},
     'idea_image': {'min_resolution': (800, 200)},
     'tileimage': {'min_resolution': (500, 300)},
+    'topic_image': {'min_resolution': (800, 200)},
```

Comment:
As long as they don't differ, you can use the same one as for the idea, IMHO.

Reply:
Yes, I thought about that and decided its clearer/less confusing to have its own..but alright, I will delete it then!

Reply:
Haha, I was lying.. now when changing it I remember that also having to change the model again made me decide to use its own :)